backend/program/scrapers/torrentio.py

```diff
@@ -63,18 +63,22 @@ def _scrape_show(self, item: MediaItem):
     def _scrape_items(self, items: list):
         amount_scraped = 0
         for item in items:
-            data = self.api_scrape(item)
-            log_string = item.title
-            if item.type == "season":
-                log_string = f"{item.parent.title} season {item.number}"
-            if item.type == "episode":
-                log_string = f"{item.parent.parent.title} season {item.parent.number} episode {item.number}"
-            if len(data) > 0:
-                item.set("streams", data)
-                logger.debug("Found %s streams for %s", len(data), log_string)
-                amount_scraped += 1
+            log_string = ""  # Initialize log_string with a default value
+            try:

```

Comment:
Dont put this whole thing under try block, handle exceptions elsewhere

Reply:
Ah I forgot to take out the try block, was using to debug